### PR TITLE
Added basic explanation and tricks for cycript

### PR DESCRIPTION
**Pull request by ryantzj**
#214

### CODE REVIEW

File: Document/0x06c-Reverse-Engineering-and-Tampering.md [hunk 1]
```diff
@@ -147,18 +147,69 @@ Attaching to process 2670...
...
+Cydia Substrate (formerly called MobileSubstrate) is the de-facto standard framework for developing run-time patches (“Cydia Substrate extensions”) on iOS. It comes with Cynject, a tool that provides code injection support for C. Cycript is a scripting language developed by Jay Freeman (saurik). Cycript injects a JavaScriptCore VM into the running process. Users can then manipulate the process using a hybrid of Objective-C++ and JavaScript syntax through the Cycript interactive console. It is also possible to access and instantiate Objective-C classes inside a running process. Some examples for the use of Cycript are listed in the iOS chapter.
 
-Cycript injects a JavaScriptCore VM into the running process. Users can then manipulate the process using JavaScript with some syntax extensions through the Cycript Console.
+Download SDK, unpack and Installation
```
Comment (commjoen) [inline Document/0x06c-Reverse-Engineering-and-Tampering.md]: should this be a header? or an introductionary sentence to the commands below?

File: Document/0x06c-Reverse-Engineering-and-Tampering.md [hunk 2]
```diff
@@ -147,18 +147,69 @@ Attaching to process 2670...
...
+We have injected cycript into SpringBoard, lets try to trigger an alert message on SpringBoard with cycript. 		
+
+	cy# alertView = [[UIAlertView alloc] initWithTitle:@"OWASP MSTG" message:@"Mobile Security Testing Guide"  delegate:nil cancelButtonitle:@"OK" otherButtonTitles:nil]
+	#"<UIAlertView: 0x1645c550; frame = (0 0; 0 0); layer = <CALayer: 0x164df160>>"
+	cy# [alertView show]
+	cy# [alertView release]
+
+![Cycript Alert Sample](Images/Chapters/0x06c/cycript_sample.png)
+
+To discover Document directory with cycript
```
Comment (sushi2k) [inline Document/0x06c-Reverse-Engineering-and-Tampering.md]: Maybe "Discover the document directory with cycript:"

File: Document/0x06c-Reverse-Engineering-and-Tampering.md [hunk 3]
```diff
@@ -147,18 +147,69 @@ Attaching to process 2670...
...
+	$ ps -ef | grep SpringBoard 
+	501 78 1 0 0:00.00 ?? 0:10.57 /System/Library/CoreServices/SpringBoard.app/SpringBoard
+	$ ./cycript -p 78
+	cy#
+We have injected cycript into SpringBoard, lets try to trigger an alert message on SpringBoard with cycript. 		
+
+	cy# alertView = [[UIAlertView alloc] initWithTitle:@"OWASP MSTG" message:@"Mobile Security Testing Guide"  delegate:nil cancelButtonitle:@"OK" otherButtonTitles:nil]
+	#"<UIAlertView: 0x1645c550; frame = (0 0; 0 0); layer = <CALayer: 0x164df160>>"
+	cy# [alertView show]
+	cy# [alertView release]
```
Comment (sushi2k) [inline Document/0x06c-Reverse-Engineering-and-Tampering.md]: Can you please format all code parts, like this:

```bash
# ls -alh
```

See also https://github.com/OWASP/owasp-mstg/blob/master/style_guide.md#code-and-shell-commands

File: Document/0x06c-Reverse-Engineering-and-Tampering.md (hunk 1 above)
Comment (ryantzj) [inline Document/0x06c-Reverse-Engineering-and-Tampering.md]: Hey @commjoen it's the description for the following commands.

**Comment by sushi2k** [comment]
Thanks Ryan. Good work 👍

**Pull request merged**